pages/recipes/[id]/index.js

```diff
@@ -0,0 +1,54 @@
+import { StyledLink } from "@/components/StyledLink";
+import { useRouter } from "next/router.js";
+import useSWR from "swr";
+import {
+  HeaderDiv,
+  StyledDetailsPageContainer,
+  StyledDetailsItem,
+  StyledItemsRow,
+} from "@/components/StyledDetailsPage.js";
+
+export default function DetailsPage() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { id } = router.query;
+  const { data: recipe, isLoading, error } = useSWR(`/api/recipes/${id}`);
+
+  if (isLoading) return <h2>Loading...</h2>;
+  if (error || !isReady) return <h2>An error occured...</h2>;
+
+  return (
+    <StyledDetailsPageContainer>
+      <HeaderDiv>
```

Comment:
If this is your Header of the Website, you should give it the `header` semantic name instead of `div`.

Reply:
just changed, thank you :)